docs/pages/preview/responsive.tsx

```diff
@@ -30,7 +22,9 @@ import { Text } from '@ag.ds-next/react/text';
 import { DocumentTitle } from '../../components/DocumentTitle';
 import { responsivePreviewQueryKeys } from '../../components/code/ResponsivePreviewLink';
 import { useGetScrollbarSizes } from '../../lib/hooks/useGetScrollbarSizes';
+import { useIsIos } from '../../../packages/react/src/combobox/utils';
 
+const isProd = !!process.env.NEXT_PUBLIC_BASE_PATH;
```

Comment:
EditPage.tsx uses

```
const isDevelop =
	!!process.env.NEXT_PUBLIC_BASE_PATH || process.env.NODE_ENV !== 'production';
```

Reply:
Is that why the PR preview is broken?

`Application error: a client-side exception has occurred (see the browser console for more information).`

```
TypeError: Failed to construct 'URL': Invalid URL
    at responsive-2bb66a6d783a6f72.js:1:3867
```